### ISSUE TITLE
consider using css custom properties across style sheets

### ISSUE THREAD

**Issue by ngwese** [monome/maiden#172]
There are many places in the existing CSS code where color values are duplicated. It would be worth collecting these up as CSS properties.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

**Comment by ryanlaws** [comment]
Hopefully this is a stupid question - do we need IE support? Because per the MDN link, it looks like CSS custom properties are not a thing in IE. I don't expect this to really be an issue for the norns user base but I figured it better to ask than assume :wink:

**Comment by ngwese** [comment]
I don't think we need IE support... It seems safe to assume modern Windows with Edge, Firefox, or Chrome.

**Comment by dansimco** [comment]
Centralised css colour variables in #215 . I tried to match existing colours as close as possible while cleaning up a bit of fragmentation so it it was low-contrast before, it still is in my PR.
